pkg/controller/volume/attachdetach/attach_detach_controller.go

```diff
@@ -809,6 +809,10 @@ func (adc *attachDetachController) GetNodeLabels() (map[string]string, error) {
 	return nil, fmt.Errorf("GetNodeLabels() unsupported in Attach/Detach controller")
 }
 
+func (adc *attachDetachController) GetNodeFields() (map[string]string, error) {
```

Comment:
We have a KubeletVolumeHost now. Can we add this method to that instead so that we don't need these unimplemented changes in the other controllers?